### PR TITLE
A Big Hunk Of Documentation

### PR DESCRIPTION
**Pull request by larsks**

### CODE REVIEW

**Review by paramite — APPROVED** [review]

File: how-to-test.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      -e /usr/share/openstack-tripleo-heat-templates/environments/monitoring-environment.yaml \
+      -e /usr/share/openstack-tripleo-heat-templates/environments/logging-environment.yaml \
+      -e params.yaml
+
+[tripleo-quickstart]: tripleo-heat-templates/environments/logging-environment.yaml
+
+## Test that things are working
+
+Once your TripleO environment has finished deploying, you should be
+able to browse to the Kibana UI at `http://192.168.1.20/kibana` (or whichever
```
Comment (jbadiapa) [inline how-to-test.md]: Currently kibana and uchiwa are both using /

File: how-to-test.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+IP belongs to `server1` in the inventory) and see log messages from
+your overcloud hosts.
+
+Similarly, you should be able to browse to `http://192.168.1.25/uchiwa` (or whichever IP belongs to `server2` in the inventory) and see the Uchiwa web UI.
```
Comment (jbadiapa) [inline how-to-test.md]: ditto

File: how-to-test.md (hunk 1 above)
Comment (mrunge) [reply how-to-test.md]: That was fixed with https://github.com/centos-opstools/opstools-ansible/pull/66

File: how-to-test.md (hunk 2 above)
Comment (mrunge) [reply how-to-test.md]: fixed by https://github.com/centos-opstools/opstools-ansible/pull/66

**Pull request closed without merging**